pyVHDLParser/Token/Parser.py

```diff
@@ -335,14 +336,7 @@ def GetVHDLTokenizer(cls, iterable: Iterator[str]):
 			elif tokenKind is cls.TokenKind.PossibleCharacterLiteral:
 				buffer += char
 				if len(buffer) == 2:
-					if buffer[1] == "'":
-						previousToken =   CharacterToken(previousToken, "'", start)
-						yield previousToken
-						previousToken =   CharacterToken(previousToken, "'", SourceCodePosition(row, column, absolute))
-						yield previousToken
-						tokenKind =       cls.TokenKind.OtherChars
-					else:
-						continue
+					continue # TODO: Merge with changes from #56!

```

Comment:
This `if`-block will be merged with the changes made by #56.

Reply:
#56 was merged. What's next for this PR? Anything I can do?

Reply:
As soon as I have some spare time I will rebase this branch. Since it touches the same `if`-branches in the parser as #56 I will have a look if something can be cleaned up.

I would appreciate your thoughts about my other comment, though.